src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -235,4 +255,10 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin) {
   color: $gray7;
   font-size: $px-16;
 }
+.dialog-text {
+  padding-bottom: 10px;
+}
+.entity-name {
+  text-transform: uppercase;
```

Comment:
Please use built-in Vuetify class in the template instead of this.

https://vuetifyjs.com/en/styles/text-and-typography/

Reply:
done